test_security/package.xml

```diff
@@ -12,7 +12,7 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
   <test_depend>ament_cmake</test_depend>
-  <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_cmake_pytest</test_depend>
```

Comment:
Nit: alphabetical order

Reply:
I didn't notice that they were out-of-order before already. Anyway fixed.